src/org/opensolaris/opengrok/authorization/AuthorizationStack.java

```diff
@@ -269,6 +275,18 @@ protected boolean processStack(Nameable entity,
                 }
             }
         }
+        time = System.currentTimeMillis() - time;
+
+        stats.addRequestTime(pluginPredicate.getRequest(),
+                String.format("authorization_in_stack_%s_%s", getName(), overallDecision ? "positive" : "negative"),
```

Comment:
I don't think this is necessary
 - we dont do this for plugins (only stacks)
 - there is no duplication in the format strings